packages/ecmascript/ecmascript-tests.js

```diff
@@ -0,0 +1,5 @@
+// Write your tests here!
+// Here is an example.
+Tinytest.add('example', function (test) {
+  test.equal(true, true);
+});
```

Comment:
Need to add real tests (perhaps in a different package, a la https://github.com/meteor/react-packages/tree/master/packages/babel-tests), or remove this file.